client/src/components/Forms/MarkdownInput.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Input, Form, Button, Typography } from 'antd';
+import { useState } from 'react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+export default function MarkdownInput(props: { [key: string]: any; notRequired?: boolean }) {
+  const { notRequired, ...otherProps } = props;
+  const [previewVisible, setPreviewVisible] = useState(false);
+  const [text, setText] = useState('');
+
+  const toogleView = () => {
+    setPreviewVisible(!previewVisible);
+  };
+
+  const link = (
+    <span style={{ marginLeft: '20px' }}>
+      <a
+        target="_blank"
+        href="https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax"
+      >
+        About markdown
+      </a>
+    </span>
+  );
+
+  return (
+    <div style={{ marginBottom: '20px' }}>
+      <div style={{ display: previewVisible ? 'none' : 'block' }}>
+        <Form.Item
+          {...otherProps}
+          name="comment"
+          label="Comment (markdown syntax is supported)"
+          rules={
+            notRequired
+              ? []
+              : [{ required: true, message: 'Please leave a detailed comment. Markdown syntax is allowed.', min: 30 }]
```

Comment:
We already have message about markdown support above, I think it's not necessary here

Reply:
Message removed. Also markdown comments previewer added to admin part of cross-check.

![Screenshot_17](https://user-images.githubusercontent.com/83158288/172023704-cc036903-d904-4922-86b1-3c5d537891d7.png)